### PR TITLE
Add enum not applicable to is living issue921

### PR DESCRIPTION
**Pull request by cjvillar**
### Release notes
For which schema is a change/update being suggested?

 An update to the donor_organism schema to add the option Not applicable to the enum for is_living

For <json_schema/type/biomaterial/donor_organism.json> schema:
 

### Reviews requested

- Need 2 Reviewers to approve because this is a  patch update
- Last Reviewer to review/approve, merge changes by soon

### CODE REVIEW

File: json_schema/type/biomaterial/donor_organism.json [hunk 1]
```diff
@@ -81,7 +81,8 @@
             "enum": [
                 "yes",
                 "no",
-                "unknown"
+                "unknown",
+		"not applicable"
             ],
             "user_friendly": "Alive at collection?",
             "example": "Should be one of: yes, no, or unknown.",
```
Comment (ESapenaVentura) [inline json_schema/type/biomaterial/donor_organism.json]: In the example, you should add the new option, so the users know it's an option now

File: json_schema/update_log.csv [hunk 2]
```diff
@@ -1 +1 @@
-Schema,Change type,Change message,Version,Date
```
Comment (ESapenaVentura) [inline json_schema/update_log.csv]: The header of the update log has to be left untouched, and the changes should be put in the 2nd row

File: json_schema/update_log.csv [hunk 3]
```diff
@@ -1 +1 @@
-Schema,Change type,Change message,Version,Date
+json_schema/type/biomaterial/donor_organism.json,patch,"added new enum not applicable to is living",,
```
Comment (malloryfreeberg) [inline json_schema/update_log.csv]: The update_log.csv file needs to have the header in it, and you just add a line below with your update information.

Check out the update_log.csv file in this PR (https://github.com/HumanCellAtlas/metadata-schema/pull/962) for an example.

Comment (ESapenaVentura) [inline json_schema/update_log.csv]: Delete "json_schema/", in the script that we have it is automatically added. Also, avoid the extension too. 

Try to write the message capitalizing the first word, and saying after the stop "Fixes #xxx."

File: json_schema/type/biomaterial/donor_organism.json [hunk 4]
```diff
@@ -81,10 +81,11 @@
...
                 "yes",
                 "no",
-                "unknown"
+                "unknown",
+		"not applicable"
             ],
             "user_friendly": "Alive at collection?",
-            "example": "Should be one of: yes, no, or unknown.",
+            "example": "Should be one of: yes, no, not applicable, or unknown.",
             "guidelines": "Enter yes if organism was alive at time of biomaterial collection, no if organism was dead, unknown if not known."
```
Comment (malloryfreeberg) [inline json_schema/type/biomaterial/donor_organism.json]: Can you add a bit in the guidelines to indicate to a contribute when they should use "not applicable"?

File: json_schema/update_log.csv (hunk 3 above)
Comment (malloryfreeberg) [reply json_schema/update_log.csv]: - Remove the `json_schem/` part at the beginning of the line. 
- Remove `.json` at the end of the schema name.
- The "Change message" part will be what is displayed in the changelog. Take a look at the changelog to see how others have formatted this string. It should be in sentence case, and it should indicate the issue that was addressed using "Fixes #921".

Comment (malloryfreeberg) [reply json_schema/update_log.csv]: @cjvillar ^

File: json_schema/update_log.csv [hunk 5]
```diff
@@ -1 +1,3 @@
 Schema,Change type,Change message,Version,Date
+type/biomaterial/donor_organism,patch,"Added not applicable to is_living. Fixes issue#921",,
+
```
Comment (malloryfreeberg) [inline json_schema/update_log.csv]: Almost there!
- Remove "issue" from the Fixes line. It needs to say "Fixes #921" in order for the value to become a link to the issue.
- Remove the last carriage return (line 3)

File: json_schema/update_log.csv [hunk 6]
```diff
@@ -1 +1,3 @@
 Schema,Change type,Change message,Version,Date
+type/biomaterial/donor_organism,patch,"Added not applicable to is_living. Fixes issue#921",,
```
Comment (ESapenaVentura) [inline json_schema/update_log.csv]: Almost! 😄 Look at PR #962 on the files changed :)

**Pull request merged**